accounting/fiscal_localizations/localizations/mexico.rst

```diff
@@ -275,44 +275,44 @@ process in Odoo, this considerations to understand the behavior are important.
    PPD, because It is the expected behavior legally required for
    "Cash payment".
 
-   **1.1. How can I generate an invoice with payment term `PUE`?**
+   **1.1. How do we generate an invoice with payment term `PUE`?**
 
    `According to the SAT documentation`_ a payment is classified as ``PUE`` if
-   the invoice was agreed to be fully payed before the 17th of the next
-   calendar month (the next month of the CFDI date), any other condition
-   will generate a ``PPD`` invoice.
+   it was agreed that the invoice would be fully payed before the 1st of the
+   next calendar month (the month after the one in the CFDI date); any other
+   condition will generate a ``PPD`` invoice.
 
-   **1.2. How can I get this with Odoo?**
+   **1.2. How is this done in Odoo?**
 
-   In order to set the appropriate CFDI payment term (PPD or PUE), you can
-   easily set it by using the ``Payment Terms`` defined in the invoice.
+   The appropriate CFDI payment term (PPD or PUE) is determined by the dates
+   and the ``Payment Terms`` that are selected in the invoice.
```

Comment:
If I'm correct, the following list is only an example to illustrate the paragraph.
```suggestion
   The appropriate CFDI payment term (PPD or PUE) is determined by the dates
   and the ``Payment Terms`` that are selected in the invoice. For example:
```